publish/index.js

```diff
@@ -88,6 +88,18 @@ export const publish = async ({
       cid.toString(), new Date()
     ])
 
+    await pgClient.query(`
+      UPDATE spark_rounds
+      SET measurement_count = $1
+      WHERE meridian_round = $2
+      ON CONFLICT (measurement_count) DO UPDATE
+      SET measurement_count =
+        spark_round.measurement_count + EXCLUDED.measurement_count
+    `, [
+      measurements.length,
+      roundIndex
```

Comment:
How do you handle smart-contract upgrades? When we deploy a new smart contract version, the round index resets back to zero.

In round-tracker, we are mapping `(address, roundIndex)` to a monotonically increasing `roundNumber` value. Maybe that's what you can use here?

Reply:
Oh, you are using the existing table `spark_rounds`. In that case, I believe you need to include the contract address in this query.

Reply:
https://github.com/filecoin-station/spark-api/pull/385/commits/d8d8d5e908f7fbf1148f8a4f36737f5d1b24ed95